v3/internal/generator/render/templates/eventdata.d.ts.tmpl

```diff
@@ -3,6 +3,7 @@
 {{- $models := (fixext $module.ModelsFile)}}
 {{- $imports := $module.Imports}}
 {{- with .Events -}}
+/// <reference types="@wailsio/runtime/global" />
```

Comment:
This should be conditional on `.LocalModulePath`, like the package.json.tmpl and tsconfig.json.tmpl.